Loop/Info.plist

```diff
@@ -4,6 +4,10 @@
 <dict>
 	<key>AppGroupIdentifier</key>
 	<string>$(APP_GROUP_IDENTIFIER)</string>
+	<key>BGTaskSchedulerPermittedIdentifiers</key>
```

Comment:
[nit] What does this change have to do with it?

Reply:
This was just Xcode reordering this `plist`... it has no effect on the build